app/views/notebooks/_notebook_info_jumbotron.slim

```diff
@@ -120,13 +120,18 @@ div.content-container
                   a.modal-activate href="#editNotebookModal" id="editNotebook" data-toggle="modal"
                     span.gear-dropdown-icon.glyphicon.glyphicon-upload aria-hidden="true"
                     | Upload new version
+                -if GalleryConfig.user_permissions.propose_review
+                  li
+                    a.modal-activate href="#proposeReviewModal" id="userProposeReview" data-toggle="modal"
+                      span.gear-dropdown-icon.fa.fa-file-code-o aria-hidden="true"
+                      | Propose for Review
                 li
                   a href="#" id="deleteNotebook"
                     span.gear-dropdown-icon.glyphicon.glyphicon-trash aria-hidden="true"
                     | Delete notebook
                     span id="deleteConfirm"
                     span id="deleteWaiting"
-              -if @user.admin?
+              -if !GalleryConfig.user_permissions.propose_review && @user.admin?
```

Comment:
My concern here is that it is marked as the admin section and other things might get thrown in here (like deprecation) and this won't even create the admin section now if users are allowed to propose their own reviews.

Reply:
Good point - maybe "Propose for review" always stays in the admin section for admins, and only appears in the user menu if propose_review is enabled and the user is not an admin?

Reply:
Probably for the best.

Reply:
I've updated this now